modules/cli/internal/processrunner/processrunner.go

```diff
@@ -52,6 +52,7 @@ type NewProcessRunnerOpts struct {
 	Env        []string
 	Args       []string
 	ReadyCheck *ReadyChecker
+	// TODO: add a StartsMinimized bool when TUI config is added
```

Comment:
I agree we need some config for this but I'm not sure this is the correct place. We can pair on this before tui config

Reply:
Actually, I guess ProcessRunner is what is passed into ProcessPane, so this is probably the place to put it!